### ISSUE TITLE
Nickserv Nick registration and identify box

### ISSUE THREAD

**Issue by TimeRider07** [Nordaaker/convos#488]
It would be really convenient for convos users to have a Nickserv nick registration box.

Also, if a user logins/changes nick to registered one, a box could pop up asking for the password instead of putting the command line for identification.

**Comment by poVoq** [comment]
I agree this would be nice. But I guess if you set up an LDAP backend for both Convos and your NICKSERV than you would not have to register separately at all?

**Comment by poVoq** [comment]
Hmm, so I have been thinking about this a bit further... I guess the LDAP option would not work, as Convos has no way to tell the IRCd that it is the same account, so the nickname would conflict?

Seems like right now it is a bit difficult to run a Convos instance that also allows people to be registered via Nickserv?

**Comment by jhthorsen** [comment]
I don't understand how LDAP is important. Wouldn't the information in "identify box" just be used as an on-connect-command?

**Comment by poVoq** [comment]
The question is: how to combine Convos accounts with those understood by Nickserv.

I guess when using Convos with an external 3rd party IRCd service it doesn't really matter as you will have to have two separate accounts anyways (unless you can do some Oauth2 magic). But on services that want to combine a IRCd with Convos it would be great to have a way to combine the logins.

**Comment by jhthorsen** [comment]
The nickserv identify password should also be in a different input box in the connection settings, under "Authentication settings". The "Password" box might change name to "Server password" and the new box could be "Identify password (Nickserv)" or something like that.
